### ISSUE TITLE
Redirect main pages directly instead of via search

### ISSUE THREAD

**Issue by SnorlaxMonster** [KevinPayravi/indie-wiki-buddy#424]
Per my suggestion in Issue #324, the Main Page of the Fandom/Fextralife wiki always redirects to the Main Page of the destination wiki. However, it is currently doing this via the search URL instead of a direct page link. Since the Main Page is definitely known to exist at that title, there shouldn't be any need to go via the search URL (unlike other pages, where it is unknown if the proposed target title exactly matches).

While MediaWiki is capable of just redirecting searches to the correct page when using the search URL, DokuWiki is not. For example, when Indie Wiki Buddy attempts to redirect from the [Severance Fandom Wiki's Main Page](https://severance-series.fandom.com/wiki/Severance_Wiki) to the[ independent wiki's Main Page](https://severance.wiki/start) (which is run on DokuWiki), you land on [this search page](https://severance.wiki/start?do=search&q=Start). The Main Page is one of the options listed there (under the display name "[Welcome](https://severance.wiki/start)"), but I don't think this is immediately obvious. It would greatly improve the user experience to just send the user directly to the Main Page.

Also, in search engines, if the link is changed to be a direct link instead of a search link, I think it would be better to have a slightly different redirection notice to "Look up [MAIN PAGE TITLE] on [WIKI NAME]". My suggestion is "Go to the Main Page of [WIKI NAME]", or maybe even just the generic "Visit [WIKI NAME] instead" that's used for non-mainspace pages.
![image](https://github.com/KevinPayravi/indie-wiki-buddy/assets/7100450/f4994b03-2e2c-4254-8a4d-c05cf0608b2a)

**Comment by KevinPayravi** [comment]
Great suggestions, thanks! It's a shame that DokuWiki's search endpoint doesn't directly take users to the page if it exists. On a tangent, I found some discussion about it on the [DokuWiki forums](https://forum.dokuwiki.org/d/18759-go-directly-to-page-from-search).

I've pushed a commit with the following:
* When IWB finds a main page for a DokuWiki wiki, the destination article title is set to an empty string. A search for empty string on DokuWiki will fortunately take the user right to the main page.
  * The reason I'm not directing the user to the barebone wiki URL is because some wikis (like YumeWiki) have multiple homepages that depend on the `destination_content_prefix`.
* Search engine button text for main pages has been updated per your suggestion:
<img width="623" alt="Screenshot 2024-01-07 at 3 50 58 AM" src="https://github.com/KevinPayravi/indie-wiki-buddy/assets/7636606/6346c44a-2dea-4b84-8296-859314956f2b">
